### ISSUE TITLE
Bug: Embedded Note Title makes editor line numbers misalign in Live-Preview

### ISSUE THREAD

**Issue by SkepticMystic** [mgmeyers/obsidian-embedded-note-titles#32]
Hey, thanks for this plugin :)

I've noticed that, in LP mode, the embedded note title pushes the rest of the note downwards, which misaligns the line numbers from their corresponding lines

![image](https://user-images.githubusercontent.com/70717676/169273881-baad2be3-9f44-4ba1-bc02-ad49849440a9.png)

In this img, the start of the yaml `---` should be at line 1, but is showing at line 4.

I had the same issue with Breadcrumbs, and used this solution to fix it: https://github.com/SkepticMystic/breadcrumbs/blob/7edc8ed25f2467da7e4810038085c58bcfe4a2cb/src/Views/TrailView.ts#L229

Basically, get the height of the embedded title, and add that to the height of the gutter
